client/package.json

```diff
@@ -15,7 +15,8 @@
     "react-dom": "^16.12.0",
     "react-openlayers": "^0.3.0",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.0"
+    "react-scripts": "3.4.0",
+    "shortid": "^2.2.15"
```

Comment:
Where do you use this library? Is it need?

Reply:
In component feedback. its generate unique key.

Reply:
Why you don't use `itme.id`? This variable name `item` isn't self documented, but I think it is feedback from backend and it has id

Reply:
I have not issue in this case. I can use two variant.

Reply:
Yes you can, but you add useless library, run function `.generate()` every time when this component is updeted and force user to download additional javascript